src/UserList.js

```diff
@@ -1,20 +1,21 @@
 import React from "react";
 import users from "./getUsers";
 
-const v = { display: "block" };
-const h = { display: "none" };
+const v = {display: "block"};
+const h = {display: "none"};
 
 export default class UserList extends React.Component {
-    state = {
-        clicked: 0
-    };
-    onClickEmitter = e => {
+    // state = {
```

Comment:
You are hacking the component here with clicked flag in state. The end result works as expected but what you are actually modifying and displaying based on "othersClicked" that is a global object from `getUsers.js`.